pycbc/distributions/sky_location.py

```diff
@@ -306,91 +224,115 @@ def from_config(cls, cp, section, variable_args):
                 "included in tag portion of section name"
             )
         healpix_file = str(cp.get_opt_tag(section, 'healpix_file', tag))
-        coverage = 0.9999
-        if cp.has_option_tag(section, 'coverage', tag):
-            coverage = float(cp.get_opt_tag(section, 'coverage', tag))
-
-        rasterization_nside = 64
-        if cp.has_option_tag(section, 'rasterization_nside', tag):
-            rasterization_nside = int(
-                cp.get_opt_tag(section, 'rasterization_nside', tag)
-            )
-        return cls(
-            healpix_file=healpix_file,
-            coverage=coverage,
-            rasterization_nside=rasterization_nside,
+        return cls(healpix_file=healpix_file)
+
+    def pixel_corners(self, indices):
+        """Return the Cartesian vectors corresponding to the corners of one or
+        more HEALPix pixels. Dimension 0 is the pixel index, 1 is the Cartesian
+        coordinate, 2 is the corner index.
+        """
+        return numpy.array(
+            [self.healpix_map.boundaries(pi, step=1) for pi in indices]
         )
 
+    def normalize_azimuth(self, phi):
+        """Helper function to ensure the azimuthal coordinate stays within
+        [0,2π).
+        """
+        phi[phi < 0] += 2 * numpy.pi
+        phi[phi >= 2 * numpy.pi] -= 2 * numpy.pi
```

Comment:
It may be safer to place these in a loop, i.e., keep adding (subtracting) 2 pi until phi is greater than 0 (less than 2 pi).

Reply:
I'm not sure whether more than one adjustment is ever needed, but it wouldn't hurt.